schema/annotation-source.yaml

```diff
@@ -25,22 +25,23 @@ $defs:
         type: string
         description: >-
           A free-text description of the InformationEntity.
-      strength:
+      confidence_level:
         $ref_curie: gks.core:Coding
         description: >-
-          A coded term describing the strength of support that the information the statement
-          represents is true.
+          A term describing the degree of confidence held by the creator 
+          of the information entity, that the information it represents is true 
+          (e.g. 'high confidence', 'likely true')
       confidence_score:
         $ref: "#/$defs/DataItem"
         description: >-
           A quantitative score reflecting the degree of confidence that the information 
           the information entity represents is true.
-      method:
+      specified_by:
         $ref: "#/$defs/Method"
         description: >-
           A :ref:`Method` that describes all or part of the process through which the information was
           generated.
-      contributions:
+      contribution:
```

Comment:
Generally I am partial toward non-plural attribute names, but happy to make an exception if people feel this is better.